src/wrappers/pubsub.js

```diff
@@ -0,0 +1,118 @@
+/**
+ * @fileoverview Handlers for Pubsub instrumentation
+ */
+
+const shimmer = require('shimmer');
+const {
+    tracer,
+    tryRequire,
+    eventInterface,
+    utils,
+} = require('epsagon');
+const traceContext = require('../trace_context.js');
+
+const subscriber = tryRequire('@google-cloud/pubsub/build/src/subscriber');
+
+/**
+ * Handle subscriber event emitter of eventName='message'
+ * @param {Message} message received message.
+ * @param {*} originalHandler listener callback function.
+ * @param {*} requestFunctionThis request arguments.
+ */
+function pubSubSubscriberMiddleware(message, originalHandler, requestFunctionThis) {
+    let originalHandlerSyncErr;
+    try {
+        // Initialize tracer and evnets.
+        tracer.restart();
+        const { slsEvent: pubSubEvent, startTime: pubSubStartTime } =
+        eventInterface.initializeEvent(
+            'pubsub',
+            requestFunctionThis.projectId,
+            'messagePullingListener',
+            'trigger'
+        );
+        tracer.addEvent(pubSubEvent);
+
+        // Getting message data.
+        let runnerMetadata = { messageId: message.id };
```

Comment:
ok now I understand. should be triggerMetadata (as it is put to the trigger)

Reply:
I should have noticed that I changed the name to the wrong name